packages/components-js/tests/e2e/specs/multi-select.e2e.ts

```diff
@@ -721,12 +721,12 @@ test.describe('keyboard and click events', () => {
 
     await page.keyboard.press('Tab');
 
-    expect(await getDropdownDisplay(page)).toBe('none');
+    await expect.poll(async () => await getDropdownDisplay(page)).toBe('none');
```

Comment:
Could use `toBeHidden` & `toBeHidden`

```suggestion
    await expect(getDropdown(page)).toBeHidden()
```